demos/projects/ESPRESSIF/esp32/components/sample-azure-iot/Kconfig.projbuild

```
@@ -1,69 +1,66 @@
-        string "Azure IoT Device ID"
-        default ""
-        help
-            "Set the Azure IoT Device ID."
-
-    config AZURE_IOT_DEVICE_SYMMETRIC_KEY
-        string "Azure IoT Device Symmetric Key"
-        default ""
-        help
-            "Set the Azure IoT Device Symmetric Key (if using SAS token authentication)."
-
-    config AZURE_IOT_MODULE_ID
-        string "Azure IoT Module ID"
-        default ""
-        help
-            "Set the Azure IoT Module ID (optional for this sample)."
-
-    config ENABLE_DPS_SAMPLE
-        bool "Enable Device Provisioning Sample"
-        default false
-        help
-            Set it to true to enable this sample to be provisioned by Azure DPS.
-
-    config AZURE_DPS_FQDN
-        string "Azure Device Provisioning Service FQDN"
-        default "global.azure-devices-provisioning.net"
-        help
-            "Set the Azure Device Provisioning Service FQDN."
-
-    config AZURE_DPS_ID_SCOPE
-        string "Azure Device Provisioning Service ID Scope"
-        default ""
-        help
-            "Set the Azure Device Provisioning Service ID Scope."
-
-    config AZURE_DPS_REGISTRATION_ID
-        string "Azure Device Provisioning Service Registration ID"
-        default ""
-        help
-            "Set the Azure Device Provisioning Service Registration ID."
-
-    config AZURE_TASK_STACKSIZE
-        int "Azure Task Stack Size"
-        default 2048
-        help
-            "Set the stack size of the main demo task."
-
-    config NETWORK_BUFFER_SIZE
-        int "MQTT packet buffer size"
-        default 5120
-        help
-            "Set the size of the network buffer for MQTT packets."
-
-endmenu
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+menu "Azure IoT middleware for FreeRTOS Main Task Configuration"
+
+    config AZURE_SAMPLE_USE_PLUG_AND_PLAY
+        bool "Use PnP in Azure Sample"
+        default true
+        help
+            Set it to true to enable this sample to use Azure Plug and Play (PnP).
+
+    config AZURE_IOT_HUB_FQDN
+        string "Azure IoT Hub FQDN"
+        default "<hostname>.azure-devices.net"
+        help
+            "Set the Azure IoT Hub FQDN."
+
+    config AZURE_IOT_DEVICE_ID
+        string "Azure IoT Device ID"
+        default ""
+        help
+            "Set the Azure IoT Device ID."
+
+    config AZURE_IOT_DEVICE_SYMMETRIC_KEY
+        string "Azure IoT Device Symmetric Key"
+        default ""
+        help
+            "Set the Azure IoT Device Symmetric Key (if using SAS token authentication)."
+
+    config AZURE_IOT_MODULE_ID
+        string "Azure IoT Module ID"
+        default ""
+        help
+            "Set the Azure IoT Module ID (optional for this sample)."
+
+    config ENABLE_DPS_SAMPLE
+        bool "Enable Device Provisioning Sample"
+        default false
+        help
+            Set it to true to enable this sample to be provisioned by Azure DPS.

```

Wasn't there some issues with using the acronym DPS from a copyright perspective?

```suggestion
            Set it to true to enable this sample to be provisioned by Azure Device Provisioning Service.

```

---

Good idea. Fixed.